1.11/deploying-services/marathon-parameters/index.md

```diff
@@ -201,17 +201,24 @@ Set up a stateful application. For more information, see [local persistent volum
+
+    - **inactiveAfterSeconds** - If an instance is unreachable for longer than `inactiveAfter` seconds it is marked as inactive. This will trigger a new instance launch. The original task is not expunged yet. Must be less than or equal to `expungeAfterSeconds`.
+    - **expungeAfterSeconds** - If an instance is unreachable for longer than `expungeAfterSeconds` it will be expunged.  That means it will be killed if it ever comes back. Instances are usually marked as unreachable before they are expunged but they don't have to. This value is required to be greater than `inactiveAfterSeconds` unless both are zero. If the instance has any persistent volumes associated with it, then they will be destroyed and associated data will be deleted.
+
 ### upgradeStrategy
 The strategy that controls when Marathon stops old versions and launches new versions. During an upgrade all instances of an application are replaced by a new version. 
 
-- **minimumHealthCapacity** The minimum percentage (expressed as a decimal fraction between `0.0` and `1.0`) of nodes that remain healthy during an upgrade. During an upgrade, Marathon ensures that this number of healthy instances are up. The default is `1.0`, which means no old instance can be stopped before another healthy new version is deployed. A value of `0.5` means that during an upgrade half of the old version instances are stopped first to make space for the new version. A value of `0` means take all instances down immediately and replace with the new application.
-- **maximumOverCapacity** The maximum percentage (expressed as a decimal fraction between `0.0` and `1.0`) of new instances that can be launched at any point during an upgrade. The default value is `1`, which means that all old and new instances can exist during the upgrade process. A value of `0.1` means that during the upgrade process 10% more capacity than usual may be used for old and new instances. A value of `0.0` means that even during the upgrade process no more capacity may be used for the new instances than usual. Only when an old version is stopped, a new instance can be deployed.
+  - **minimumHealthCapacity** The minimum percentage (expressed as a decimal fraction between `0.0` and `1.0`) of nodes that remain healthy during an upgrade. During an upgrade, Marathon ensures that this number of healthy instances are up. The default is `1.0`, which means no old instance can be stopped before another healthy new version is deployed. A value of `0.5` means that during an upgrade half of the old version instances are stopped first to make space for the new version. A value of `0` means take all instances down immediately and replace with the new application.
```

Comment:
The number of spaces at the beginning of a paragraph or list item seems to be inconsistent. Also, it seems there should be a dash after the entity being defined, according to the way the other definitions in this file are formatted. What do you think?